docs/cse/match-lists-suppressed-lists/standard-match-lists.md

```diff
@@ -147,20 +114,40 @@ none
 
 The following CSE rules refer to this Match List:
 
-* AWS_CloudTrail_Aggressive_Reconnaissance.json
-* AWS_CloudTrail_Database_Snapshot_Created.json
-* AWS_CloudTrail_GetSecretValue_from_non_Amazon_IP.json
-* AWS_CloudTrail_Reconnaissance_related_event.json
-* AWS_CloudTrail_sensitive_activity_in_KMS.json
-* AWS_ECS_Cluster_Deleted.json
-* AWS_Route_53_Domain_Registered.json
-* AWS_Route_53_Reconnaissance.json
-* AWS_Route_53_Service_Tampering.json
-* AWS_Route_53_TestDNSAnswer.json
-* AWS_Route_53_Traffic_Policy_Creation.json
-* AWS_WAF_Rule_Group_Updated.json
-* AWS_WAF_Rule_Updated.json
-* Anomalous_AWS_User_Executed_a_Command_on_ECS_Container.json
+* AWS Cloud Storage Deletion
+* AWS CloudTrail - Aggressive Reconnaissance
+* AWS CloudTrail - Database Snapshot Created
+* AWS CloudTrail - GetSecretValue from non Amazon IP
+* AWS CloudTrail - Reconnaissance related event
+* AWS CloudTrail - sensitive activity in KMS
+* AWS CloudWatch Alarm Actions Disabled
+* AWS CloudWatch Alarm Deletion
+* AWS CloudWatch Anomaly Detector Deletion
+* AWS CloudWatch Log Group Deletion
+* AWS CloudWatch Log Stream Deletion
```

Comment:
Should be Amazon CloudWatch [per their branding](https://aws.amazon.com/documentation-overview/)

Reply:
As John mentioned, these are the names of the Rules in CloudSIEM, so I would recommend our threat labs team changes the names of the rules to align with their branding.